db/fixtures/miq_user_roles.yml

```diff
@@ -1520,6 +1520,9 @@
   - embedded_automation_manager_credentials_view
   - embedded_configuration_script_payload_view
   - embedded_configuration_script_source_view
+  - embedded_workflow_view
+  - embedded_repositories_view
```

Comment:
Minor, and I noticed this after merge, but this and credentials should be singular like the other routes

Reply:
Yeah I see _some_ are plural but definitely the minority.  `embedded_repository_view` looks right